hash/hash.go

```diff
@@ -58,6 +59,21 @@ func Int(h int) Hash {
 	return hash
 }
 
+// Random returns a new random hash.
+func Random() (Hash, error) {
```

Comment:
Should use this function is way more test ;)

Reply:
Int is fine :P